src/codewhisperer/util/supplementalContext/crossFileContextUtil.ts

```diff
@@ -68,7 +68,10 @@ export async function fetchSupplementalContextForSrc(
         })
     }
 
-    return supplementalContexts
+    // DO NOT send code chunk with empty content
```

Comment:
You can check this BEFORE you push anything into this supplementalContexts list. Doing it in the end will run the for loop twice.

Reply:
yea make sense, but I prefer keeping it for now so that we have a similar place/structure filter out blank strings between 2 implementation. Plus, the size of the code chunk is constant (1~3), the runtime shouldn't be an issue.